Tests/ExampleAnalyzers/RoslynAnalyzer10/RoslynAnalyzer10/RoslynAnalyzer10.csproj

```diff
@@ -1,113 +1,38 @@
-    </Reference>
-    <Reference Include="System.Composition.AttributedModel">
-      <HintPath>..\..\..\..\packages\Microsoft.Composition.1.0.27\lib\portable-net45+win8+wp8+wpa81\System.Composition.AttributedModel.dll</HintPath>
-    </Reference>
-    <Reference Include="System.Composition.Hosting">
-      <HintPath>..\..\..\..\packages\Microsoft.Composition.1.0.27\lib\portable-net45+win8+wp8+wpa81\System.Composition.Hosting.dll</HintPath>
-    </Reference>
-    <Reference Include="System.Composition.Runtime">
-      <HintPath>..\..\..\..\packages\Microsoft.Composition.1.0.27\lib\portable-net45+win8+wp8+wpa81\System.Composition.Runtime.dll</HintPath>
-    </Reference>
-    <Reference Include="System.Composition.TypedParts">
-      <HintPath>..\..\..\..\packages\Microsoft.Composition.1.0.27\lib\portable-net45+win8+wp8+wpa81\System.Composition.TypedParts.dll</HintPath>
-    </Reference>
-    <Reference Include="System.Reflection.Metadata, Version=1.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\..\..\packages\System.Reflection.Metadata.1.1.0\lib\portable-net45+win8\System.Reflection.Metadata.dll</HintPath>
-    </Reference>
-  </ItemGroup>
-  <ItemGroup>
-    <AdditionalFiles Include="..\..\..\..\SonarLint.xml">
-      <Link>Properties\SonarLint.xml</Link>
-    </AdditionalFiles>
-  </ItemGroup>
-  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\v4.5\Microsoft.Portable.CSharp.targets" />
-</Project>
\ No newline at end of file
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net45</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.CodeAnalysis" Version="1.0.0"/>
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="1.0.0"/>
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.0.0"/>
+        <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic" Version="1.0.0"/>
+        <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces" Version="1.0.0"/>
+        <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="1.0.0" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <None Include="Diagnostic.nuspec">
+            <SubType>Designer</SubType>
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
+        <None Include="tools\install.ps1">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
+        <None Include="tools\uninstall.ps1">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
+        <None Include="ReadMe.txt"/>
+        <None Remove="packages.lock.json" />
+    </ItemGroup>
+    
+    <ItemGroup>
+        <AdditionalFiles Include="..\..\..\..\SonarLint.xml">
```

Comment:
Such file doesn't exist

Reply:
It does exist at the root of the repository.
It may be removed.
I will look into it

Reply:
Does not seems to be needed.
I will remove it.